Frontend/src/test/pages/dashboard/components/Customer/EditAddressDialog.test.jsx

```
@@ -0,0 +1,116 @@
+
+describe('<EditAddressDialog />', () => {
+  const sampleAddress = {
+    id:      42,
+    name:    'Home',
+    street:  '123 Main',
+    city:    'Testville',
+    state:   'TS',
+    zip:     '12345',
+    country: 'Countryland',
+  }
+  let onOpenChange, onSave
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)